### PR TITLE
Add email connector docs

### PR DESCRIPTION
**Pull request by dinuish94**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+# Email Connector Example
+
+Email Connector can be used to perform operations using standard protocols such as SMTP, IMAP and POP3. 
+
+## What you'll build
+
+This example explains how to use Email Connector to send an email and retrieve the same email from GMail. The user sends a payload with the recipients, content and attachments of the email. Then, by invoking another API resource, the content of the sent email will be retrieved.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md:7]: from GMail --> using [Gmail](https://www.google.com/gmail/)

File: en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
+# Email Connector Example
+
+Email Connector can be used to perform operations using standard protocols such as SMTP, IMAP and POP3. 
+
+## What you'll build
+
+This example explains how to use Email Connector to send an email and retrieve the same email from GMail. The user sends a payload with the recipients, content and attachments of the email. Then, by invoking another API resource, the content of the sent email will be retrieved. 
+
+It will have two HTTP API resources, which are `send` and `retrieve`.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md:9]: The example consists of an API named as `EmailConnector API` with two resources `send` and `retrieve`.

File: en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+
+Email Connector can be used to perform operations using standard protocols such as SMTP, IMAP and POP3. 
+
+## What you'll build
+
+This example explains how to use Email Connector to send an email and retrieve the same email from GMail. The user sends a payload with the recipients, content and attachments of the email. Then, by invoking another API resource, the content of the sent email will be retrieved. 
+
+It will have two HTTP API resources, which are `send` and `retrieve`. 
+
+* `/send `: The user sends the request payload which includes the recipients, content and attachments of the email. This request is sent to WSO2 EI by invoking the EmailConnector API. It will send the email to the relevant recipients.
```
Comment (abeykoon) [inline en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md:11]: `/send `: This API resource will retrieve information from the incoming HTTP post request such as recipients, content and attachments and construct the email and send to the mentioned recipients.

File: en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md (hunk 1 above)
Comment (dinuish94) [reply en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md]: Fixed

File: en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md (hunk 3 above)
Comment (dinuish94) [reply en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md:11]: Added to the section where the API resource is being added

File: en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+10. Create the next API resource, which is `/retrieve` by dragging and dropping another API resource to the design view. This will be used to retrieve the email we just sent. This will also be a `POST` request.
+TODO:<img>
+
+11. Drag and drop the 'list' operation of the Email Connector to the Design View as shown below.
+
+12. In this operation we are going to receive following inputs from the user. 
+    - subjectRegex - Subject Regex to filter the email from.
+    
+13. We will next iterate the response received and obtain the email content of each email using the `getEmailBody` operation. In order to do this, drag and drop the 'foreach' mediator as shown below and enter `//emails/email` as the Foreach Expression in the properties window.
```
[the remark this answers is not in the document]
Comment (dinuish94) [reply en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md]: Fixed

File: en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md (hunk 2 above)
Comment (dinuish94) [reply en/micro-integrator/docs/references/connectors/email-connector/email-connector-example.md]: Fixed

**Pull request merged**